components/network/ntp/patches/60-rem-vpath_after-ntpd.patch

```diff
@@ -0,0 +1,11 @@
+--- ntpd/Makefile.am.orig	2023-05-31 12:31:38.000000000 +0200
```

Comment:
Please add a comment explaining why we do need this.  Thank you.

Reply:
removed